app/src/main/java/org/oppia/android/app/classroom/topiclist/TopicCard.kt

```diff
@@ -0,0 +1,98 @@
+package org.oppia.android.app.classroom.topiclist
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.oppia.android.R
+import org.oppia.android.app.classroom.getDrawableResource
+import org.oppia.android.app.home.topiclist.TopicSummaryViewModel
+
+/** Displays a card with the topic summary information. */
+@Composable
+fun TopicCard(topicSummaryViewModel: TopicSummaryViewModel) {
+  Card(
+    modifier = Modifier
+      .padding(
+        start = dimensionResource(R.dimen.promoted_story_card_layout_margin_start),
+        end = dimensionResource(R.dimen.promoted_story_card_layout_margin_end),
+      )
+      .clickable { topicSummaryViewModel.clickOnSummaryTile() },
+    elevation = 4.dp,
+  ) {
+    Column(
+      verticalArrangement = Arrangement.Center,
+    ) {
+      Image(
+        painter = painterResource(
+          id = topicSummaryViewModel.topicSummary.topicThumbnail.getDrawableResource()
+        ),
+        contentDescription = "${topicSummaryViewModel.title} Card",
```

Comment:
Could potentially be more descriptive. `thumbnailGraphic.name` should possibly give the name of the graphic as per enum eg BAKER.
```suggestion
      contentDescription = "Picture of a ${topicSummaryViewModel.topicSummary.topicThumbnail.thumbnailGraphic.name}.",
```

Reply:
Done.